subprojects/tooling-api/src/integTest/groovy/org/gradle/integtests/tooling/r33/BuildProgressCrossVersionSpec.groovy

```diff
@@ -263,7 +263,6 @@ class BuildProgressCrossVersionSpec extends ToolingApiSpecification {
 
         then:
         !events.operations.find { it.name.contains("compileClasspath") }
-        events.operation("Run tasks").children.empty
```

Comment:
I think this is trying to show that when you don't ask for task events, you don't see the children of the tasks:

e.g., the events would have looked like this before 4.0:
Run Tasks -> Task :compileJava -> Resolve dependencies :compileClasspath

And if you didn't want the task events, you shouldn't see the resolve events. I think the `!events.operations.find { it.name.contains("compileClasspath") }` covers the important part of this check (that we don't see the resolve events).